Scripts/script-CommonServer.yml

```diff
@@ -23,6 +23,31 @@ script: |-
   }
 
   /**
+   * Clean an object from empty fields
+   * @returns {Object} the clean object
+   */
+  var cleanObject = function(obj) {
+        var cleanSingle = function(contents) {
+            var cleanContents = {};
+            var keys = Object.keys(contents);
+            for (var i = 0; i < keys.length; i++) {
+                if (contents[keys[i]] || contents[keys[i]] === false) {
+                    cleanContents[keys[i]] = contents[keys[i]];
+                }
+            }
+            return cleanContents;
+        }
+        if (obj instanceof Array) {
```

Comment:
what if you have an array of objects inside the root.
You are not going recursively here